src/index.js

```diff
@@ -38,6 +38,10 @@ export const getLocale = options => request => {
   return getSupported(getLanguage(getFromRequest(request)))
 }
 
+export const getSupportedLanguage = options => localeString => {
+  return getSupportedLocale(options)(getLanguage(localeString))
+}
```

Comment:
Ah. It sucks that we’d have to pass the same config everytime we call this function. What we really want is to simply pass a localeString and get the language back, maybe even without having to make two calls because of the closure… 😅